library/Icinga/Crypt/RSA.php

```diff
@@ -0,0 +1,203 @@
+<?php
+/* Icinga Web 2 | (c) 2020 Icinga Development Team | GPLv2+ */
+
+namespace Icinga\Crypt;
+
+use InvalidArgumentException;
+use UnexpectedValueException;
+
+/**
+ * Class RSA
```

Comment:
The first line of a class' or method's PHPDoc is the first thing what's read when looking at API docs and should summarise with as less words as possible what the class or method is doing.

I'd prefer something like the following here:

> Data encryption and decryption using RSA keys also supporting key generation

Then you may remove the following extra description:

> This class allows to encrypt and decrypt data.